src/domain/admin/organization/AdminOrganizationsRoutes.tsx

```diff
@@ -7,18 +7,30 @@ import { Error404, PageProps } from '../../../pages';
 import { nameOfUrl } from '../../../routing/url-params';
 import OrganizationAdminRoutes from './OrganizationAdminRoutes';
 import AdminOrganizationsPage from '../../../pages/Admin/AdminOrganizations/AdminOrganizationsPage';
+import { useTranslation } from 'react-i18next';
+import AdminLayout from '../toplevel/AdminLayout';
+import { AdminSection } from '../toplevel/constants';
 
 const AdminOrganizationsRoutes: FC<PageProps> = ({ paths }) => {
   const { pathname: url } = useResolvedPath('.');
   const currentPaths = useMemo(() => [...paths, { value: url, name: 'organizations', real: true }], [paths]);
+  const { t } = useTranslation();
 
   return (
     <Routes>
       <Route path={'/'}>
         <Route index element={<AdminOrganizationsPage paths={currentPaths} />}></Route>
         <Route
           path={'new'}
-          element={<OrganizationPage title={'Create organization'} mode={EditMode.new} paths={currentPaths} />}
+          element={
```

Comment:
This has nothing to do, it's just a missing topBanner that I found in the administration